packages/relgen-core/src/services/context/remote/github.ts

```diff
@@ -511,7 +513,7 @@ export const githubContextService = (
             )
             .filter((file) => file.filename),
           async (file) => {
-            if (excludedContexts?.has('file-content')) {
```

Comment:
The use of 'trim' is not allowed as per the rules. Please find an alternative method to handle whitespace.